test/packages/caver.account.accountKey.js

```diff
@@ -0,0 +1,594 @@
+/*
+    Copyright 2020 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const _ = require('lodash')
+const chai = require('chai')
+const sinonChai = require('sinon-chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+chai.use(sinonChai)
+
+const expect = chai.expect
+
+const testRPCURL = require('../testrpc')
+
+const Caver = require('../../index.js')
+const utils = require('../../packages/caver-utils')
+
+const AccountKeyLegacy = require('../../packages/caver-account/src/accountKey/accountKeyLegacy')
+const AccountKeyPublic = require('../../packages/caver-account/src/accountKey/accountKeyPublic')
+const AccountKeyFail = require('../../packages/caver-account/src/accountKey/accountKeyFail')
+const AccountKeyWeightedMultiSig = require('../../packages/caver-account/src/accountKey/accountKeyWeightedMultiSig')
+const AccountKeyRoleBased = require('../../packages/caver-account/src/accountKey/accountKeyRoleBased')
+
+const { ACCOUNT_KEY_TAG } = require('../../packages/caver-account/src/accountKey/accountKeyHelper')
+
+let caver
+
+beforeEach(() => {
+    caver = new Caver(testRPCURL)
+})
+
+function isAccountKey(accountKey, expectedAccountKeyType, options = {}) {
```

Comment:
```suggestion
function testAccountKey(accountKey, expectedAccountKeyType, options = {}) {
```

The prefix `is` is somewhat misleading.